client/post-editor/editor-ground-control/test/index.jsx

```diff
@@ -45,12 +45,12 @@ describe( 'EditorGroundControl', function() {
 		mockery.registerMock( 'components/post-schedule', EmptyComponent );
 		EditorGroundControl = require( '../' ).WrappedComponent;
 
-		EditorGroundControl.prototype.__reactAutoBindMap.translate = i18n.translate;
+		EditorGroundControl.prototype.translate = i18n.translate;
 		EditorGroundControl.prototype.__reactAutoBindMap.moment = i18n.moment;
 	} );
 
 	after( function() {
-		delete EditorGroundControl.prototype.__reactAutoBindMap.translate;
+		delete EditorGroundControl.prototype.translate;
 		delete EditorGroundControl.prototype.__reactAutoBindMap.moment;
```

Comment:
Hehe, yeah. I put that change it into the React 15 upgrade branch, https://github.com/Automattic/wp-calypso/pull/5116/commits/0d123da392c19230eea570875d83f1af306f2b31